sdk/mobile-center-crashes/src/androidTest/java/com/microsoft/azure/mobile/crashes/CrashesAndroidTest.java

```diff
@@ -312,7 +309,7 @@ public void run() {
         assertEquals(2, ErrorLogHelper.getErrorStorageDirectory().listFiles().length);
 
         /* Disable. */
-        Crashes.setEnabled(false);
+        Crashes.setEnabled(false).get();
```

Comment:
it's just additional testing to make sure there is no deadlock or any issue by doing that as well.